src/Pages/Diagnostic/index.jsx

```diff
@@ -0,0 +1,5 @@
+export const DiagnosticComponent = () => {
```

Comment:
This should be importing `useParams` like the Display/index.jsx page so that you can add the `key` from the path.